lib/report/index.html.ejs

```diff
@@ -1,742 +1,723 @@
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2"
+    crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.min.js"
-          integrity="sha384-v7eExOYhwaHa3+GhP+lHytJsMcidazNdjiaggRhdbvVTVTCjweLpa23t37ZKxaCf"
-          crossorigin="anonymous"></script>
+    integrity="sha384-v7eExOYhwaHa3+GhP+lHytJsMcidazNdjiaggRhdbvVTVTCjweLpa23t37ZKxaCf"
+    crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.min.js"
-          integrity="sha384-H6KKS1H1WwuERMSm+54dYLzjg0fKqRK5ZRyASdbrI/lwrCc6bXEmtGYr5SwvP1pZ"
-          crossorigin="anonymous"></script>
+    integrity="sha384-H6KKS1H1WwuERMSm+54dYLzjg0fKqRK5ZRyASdbrI/lwrCc6bXEmtGYr5SwvP1pZ"
+    crossorigin="anonymous"></script>
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.2.0/ace.js" charset="utf=8"
-          integrity="sha384-5wv3YDwWXIxqoY02AhdpzGtKzSx7IoLw86ObEolywSYRKXx9E4rqmrU2R29LNIjR"
-          crossorigin="anonymous"></script>
+    integrity="sha384-5wv3YDwWXIxqoY02AhdpzGtKzSx7IoLw86ObEolywSYRKXx9E4rqmrU2R29LNIjR"
+    crossorigin="anonymous"></script>
 
-  <style>
-  body, p, li {
-    font-family: sans-serif;
-    font-size: 10pt;
-  }
-
-  h1 {
-    font-size: 18pt;
-    font-weight: bold;
-  }
-
-  #errorsContainer > p {
-    color: #009900;
-  }
-
-  #footer {
-    padding: 1em;
-    font-size: 0.8em;
-    color: #999;
-    text-align: center;
-    margin-top: 1em;
-    background-color: #eee;
-  }
-  #editor {
-    font-size: 8pt;
-    height: 20em;
-  }
-
-  .chart-container {
-    margin: 20px 0px;
-  }
-  </style>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous" />
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2"
+    crossorigin="anonymous"></script>
+
+  <link href="https://artilleryio.github.io/artillery-tailwind/src/build.css" rel="stylesheet" type="text/css"
+    crossorigin="anonymous" />
 
   <style>
-    h1 {
-      line-height: 200%;
+    html,
+    body,
+    p,
+    li {
+      --bg-dark: #161616;
+      --bg: #202020;
+      --fg: #2b2b2b;
+      --text: #ffffff;
+      --text-faded: #9b9b9b;
+      --border: #383838;
+      --green: #24eb70;
+      --yellow: #ffd600;
+      --red: #ff0000;
+
+      font-family: sans-serif;
+      font-size: 10pt;
```

Comment:
In general this is now conflicting with Tailwind, we have 3 competing stylesheets here. Tailwind, Bootstrap and these custom ones. Feel free to remove custom + bootstrap as it'll just make life a little more difficult 🙈

Reply:
Oh and the `font-size: 10pt` will effect everything meaning our font sizes will render different as inline styling takes precedent, so the base font size will be dictated from here.